SPECS/hyperv-daemons/hyperv-daemons.spec

```diff
@@ -27,7 +27,6 @@ Source102:      hypervvss.rules
 # HYPERV FCOPY DAEMON
 Source201:      hypervfcopyd.service
 Source202:      hypervfcopy.rules
-Patch0:         0001-clocksource-drivers-hyper-v-Re-enable-VDSO_CLOCKMODE.patch
 Patch1:         0002-add-linux-syscall-license-info.patch
```

Comment:
Nit: Should the Patch1 now become Patch0? Ditto for all other kernel specs. 

E.g. 
-Patch1:         0002-add-linux-syscall-license-info.patch
+Patch0:         0002-add-linux-syscall-license-info.patch